#2020/01/30 Patch has not yet been accepted upstream
From 51ee0c90068ef7d88a23d86579cc1f73c2f6055f Mon Sep 17 00:00:00 2001
From: Eric Timmons <etimmons@mit.edu>
Date: Sat, 1 Feb 2020 17:33:38 -0500
Subject: [PATCH 2/5] Do not require _CS_GNU_LIBPTHREAD_VERSION at runtime
 start

If there is no content in confstr for _CS_GNU_LIBPTHREAD_VERSION (i.e., on a
non GNU libc) then assume the thread implementation is good enough and let the
user deal with any fallout.
---
 src/runtime/linux-os.c | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/src/runtime/linux-os.c b/src/runtime/linux-os.c
index 89244e793..0ff55653d 100644
--- a/src/runtime/linux-os.c
+++ b/src/runtime/linux-os.c
@@ -186,8 +186,13 @@ isnptl (void)
       if (strstr (buf, "NPTL")) {
           return 1;
       }
+      return 0;
+  }
+  else {
+      /* This libc does not expose the libpthread version. Just assume we have
+       * a good enough thread implementation. */
+      return 1;
   }
-  return 0;
 }
 #endif
 
-- 
2.25.0

